### PR TITLE
refactor of blog image src

### PR DESCRIPTION
**Pull request by ravin1227**

### CODE REVIEW

File: content/blog/best-online-form-builders.md [hunk 1]
```diff
@@ -34,7 +35,7 @@ Let’s take a look at the best online form builders currently and find out how
 
 [Formester](https://formester.com/) is the one and only combined solution for online forms and HTML form backend (more on this later). The most critical aspect of any online form builder is the ease and convenience that it offers – and our platform ensures that. Creating a form for your surveys, feedback, registration, lead generation, events and a number of other applications, is comprehensively simple. The drag-and-drop interface helps you build your online form in minutes so that you can focus on efficient distribution and maximize your ROI. Further, our solution enables you to manage the collected data and access the form analytics for a detailed study of your responses.
 
-![TEDx organiser Thibaud Martinez saying that he enjoys the clean and straight forward nature of formester](/img/best-online-form-builder__formester-testimonial.png)
+![TEDx organiser Thibaud Martinez saying that he enjoys the clean and straight forward nature of formester](/blog/best-online-form-builders/formester-testimonial.jpg 'TEDx organiser Thibaud Martinez saying that he enjoys the clean and straight forward nature of formester')
```
Comment (piyushsinghania) [inline content/blog/best-online-form-builders.md:38]: Is the image path correct here? @ravin1227 
I think it should be `/img/best-online-form-builder__formester-testimonial.jpg`

File: content/blog/best-online-form-builders.md [hunk 2]
```diff
@@ -78,15 +79,15 @@ However, Google Forms do not enable full customization, and there’s no option
 
 ### 4. Paperform:
 
-![Paperform landing page](/img/best-online-form-builder__paperform.png)
+![Paperform landing page](/blog/best-online-form-builders/paperform.png 'Paperform landing page')
```
Comment (piyushsinghania) [inline content/blog/best-online-form-builders.md:82]: Here also, or am I missing something?
I see the `/blog` pattern many places.

**Review by piyushsinghania — COMMENTED** [review]
I also don't see the screenshot images with the background? Are those changes not pushed in these branch.

File: content/blog/best-online-form-builders.md (hunk 2 above)
Comment (ravin1227) [reply content/blog/best-online-form-builders.md:82]: I haven't changed all the images I will be doing this through UI I made few changes just to check

Comment (ravin1227) [reply content/blog/best-online-form-builders.md:82]: If u will check the featured blogs 1st screenshot u will see the changes

Comment (ravin1227) [reply content/blog/best-online-form-builders.md:82]: And these changes have been made during branch conflict I updated the master branch and merged soo i got updated with the master

**Review by ankitsinghaniyaz — APPROVED** [review]

**Pull request merged**